extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -2896,6 +2897,57 @@ describes.repeated(
             }
           );
         });
+
+        describe('Form Dirtiness State', () => {
```

Comment:
Deliberately only wrote tests that cover the most critical flows, while the unit tests for `form-dirtiness` are more comprehensive.